Documentation/ReleaseNotes.md

```diff
@@ -7,6 +7,12 @@
 
 ### What's new
 
+**Oculus Quest XRSDK support**
```

Comment:
I see 
![image](https://user-images.githubusercontent.com/3580640/85644203-29e69080-b64b-11ea-87c1-06bed7b5fd1b.png)
get created when I import the Oculus package. Are you seeing them too? We should probably check them in.

Reply:
Yeah I also see the loaders. Didn't include them initially but now I see we have ARCore and WMR loaders already in the project, so added these in as well